website/src/controllers/api-v2/user.js

```diff
@@ -319,9 +319,11 @@ let requiresPurchase = {
 
 let checkPreferencePurchase = (user, path, item) => {
   let itemPath = `${path}.${item}`;
-  let isDefaultPreference = _.get(shared.content.defaultAppearancePreferences, itemPath);
+  let appearance = _.get(shared.content.appearances, itemPath);
 
-  if (isDefaultPreference) return true;
+  if (appearance) {
```

Comment:
I think part of the defaults should be to set the price to 0 if it does not exist and this should now be:

```js
if (!appearance) return false;
if (appearance.price === 0) return true;
```

IE, if the appearance is not valid, return false early. If the price is 0, return true early.